spring-cloud-dataflow-docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -126,10 +126,10 @@ By default, the application registry is empty.
 As an example, register two applications, `http` and `log`, that communicate by using RabbitMQ.
 +
 ```
-dataflow:>app register --name http --type source --uri maven://org.springframework.cloud.stream.app:http-source-rabbit:1.2.0.RELEASE
+dataflow:>app register --name http --type source --uri maven://org.springframework.cloud.stream.app:http-source-rabbit:1.3.1.RELEASE
 Successfully registered application 'source:http'
 
-dataflow:>app register --name log --type sink --uri maven://org.springframework.cloud.stream.app:log-sink-rabbit:1.1.0.RELEASE
+dataflow:>app register --name log --type sink --uri maven://org.springframework.cloud.stream.app:log-sink-rabbit:1.3.1.RELEASE
```

Comment:
Updating this messes up the next getting started section for upgrading versions of the application with skipper....